Packages/com.unity.inputsystem/InputSystem/Plugins/PlayerInput/PlayerInput.cs

```diff
@@ -989,11 +1110,18 @@ public void SwitchCurrentActionMap(string mapNameOrId)
         }
 
         /// <summary>
-        /// Return the Nth player.
+        /// Return the player with specified player index.
         /// </summary>
-        /// <param name="playerIndex">Index of the player to return.</param>
+        /// <param name="playerIndex">The index into the active player list.</param>
         /// <returns>The player with the given player index or <c>null</c> if no such
         /// player exists.</returns>
+        /// <remarks>
+        /// <example>
+        /// <code>
+        /// PlayerInput player = PlayerInput.GetPlayerByIndex(0);
```

Comment:
Are we ok with non-compilable examples? @duckets

Reply:
It should compile but not in isolation. I'll await doc writer feedback